docs/examples/sensormanagement/bayesian_search_example_2.py

```diff
@@ -0,0 +1,465 @@
+# can then be given to a :class:`~.SensorManager`. The sensor manager chooses the best action(s)
+# for the current timestep. We define the platform's initial position, and limit its movement to a
+# step of three grid cells in a single direction.
+
+
+from stonesoup.movable.grid import NStepDirectionalGridMovable
+from stonesoup.platform import Platform
+from stonesoup.types.state import State, StateVector
+
+initial_loc = StateVector([[4.], [4.]])
+initial_state = State(initial_loc, start_time)
+
+platform = Platform(
+    movement_controller=NStepDirectionalGridMovable(states=[initial_state],
+                                                    position_mapping=(0, 1),
+                                                    n_steps=1,
+                                                    step_size=3,
+                                                    action_mapping=(0, 1),
+                                                    ))
+
+
+# %%
+# Creating the Sensor
+# ~~~~~~~~~~~~~~~~~~~
+# Next we make a sensor which will be mounted to the platform. In this case we use a
+# :class:`~.RadarRotatingBearingRange` sensor. The sensor has a 360 degree field of view, with a
+# range of 1.6 grid cells.
+#
+# As before, we set the probability of detection to 0.9.
+
+
+from stonesoup.sensor.radar.radar import RadarRotatingBearingRange
+
+mounted_sensor = RadarRotatingBearingRange(
+    position_mapping=(0, 2),
+    noise_covar=np.array([[np.radians(0.5) ** 2, 0],
+                          [0, 1 ** 2]]),
+    ndim_state=4,
+    rpm=0,
+    fov_angle=np.radians(360),
+    dwell_centre=StateVector([0.0]),
+    max_range=1.6,
+    resolution=np.radians(360))
+
+mounted_sensor.timestamp = start_time
+prob_det = 0.9
+
+platform.add_sensor(mounted_sensor)
+
+
+# %%
+# Creating the Search Grid
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# We will now create the environment with which our sensor and platform will interact.
+#
+# To begin, we create a 15x15 grid. As in the previous tutorial, we will do this using
+# :class:`~.ParticleState` objects, whose properties can be used to represent a position and
+# probability of target presence.
+
+
+# creating the positions of the particles that will form our search grid
+ymin = 0
+xmin = 0
+ymax = 14
+xmax = 14
+nx = 15
+ny = 15
+
+xarray = np.linspace(xmin, xmax, nx)
+yarray = np.linspace(ymin, ymax, ny)
+x, y = np.meshgrid(xarray, yarray)
+
+x_pos = x.flatten()
+y_pos = y.flatten()
+
+# %%
+# In setting the particle weights here, we are defining our initial probability distribution. In
+# this case, the initial distribution will be Gaussian, centred around (x=7, y=8).
+
+
+from stonesoup.types.state import ParticleState
+from stonesoup.types.array import StateVectors
+from scipy.stats import multivariate_normal
+
+# parameterise our prior probability distribution
+mu = np.array([7, 8])  # mean(x, y)
+sigma = np.array([[10, 0], [3, 6]])  # covariance matrix [[var(x), covar(xy)], [covar(xy), var(y)]]
```

Comment:
covar(xy) should equal covar(yx). Doesn't the following line throw an error?

Reply:
No because according to SciPy.stats the symmetry is not checked and only the lower triangle is used. No excuse not to correct it though.